LanguageExt.Core/DataTypes/Resource/Resource.Prelude.cs

```diff
@@ -0,0 +1,22 @@
+
+using LanguageExt.ClassInstances;
+using LanguageExt.TypeClasses;
+using static LanguageExt.TypeClass;
+using System;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace LanguageExt
+{
+    public static partial class Prelude
+    {
+        public static Resource<R> Use<R>(Func<R> acquire) where R : IDisposable =>
```

Comment:
Is this supposed to be lower-case? (`use`)

Reply:
I put it uppercase because it's a type constructor like `Left`, `Fail`, or `Some`.  Maybe `release` below should be uppercase as well?

Reply:
Gah!  Sorry, of course, you're right 👍